src/abstract-classes/browser-plugin.ts

```diff
@@ -21,11 +21,14 @@ export interface CommonLibrary {
     name?: () => string;
 }
 
-/**
- * @internal
- */
+/** @internal */
 export interface CommonBrowser {
-    newPage(...args: unknown[]): unknown;
+    newPage(...args: unknown[]): Promise<CommonPage>;
+}
+
+/** @internal */
+export interface CommonPage {
```

Comment:
Can't this be? I'm don't know much about the internals yet so just asking :)

```ts
import { Page as PuppeteerPage } from 'puppeteer';
import { Page as PlaywrightPage } from 'playwright';

export type CommonPage = PuppeteerPage | PlaywrightPage;
```

Reply:
Could it, yes, but since either dependencies can be missing at runtime, I think it'd widen to `any` which is... not helpful at all

Reply:
Woah, you're right.

Reply:
It's a pity they don't export just the types as a separate package... That's the con of big projects written in TS.